homeassistant/components/transport_nsw/sensor.py

```diff
@@ -132,10 +139,10 @@ def __init__(self, stop_id, route, destination, api_key):
         self._api_key = api_key
         self.info = {
             ATTR_ROUTE: self._route,
-            ATTR_DUE_IN: "n/a",
-            ATTR_DELAY: "n/a",
-            ATTR_REAL_TIME: "n/a",
-            ATTR_DESTINATION: "n/a",
+            ATTR_DUE_IN: None,
+            ATTR_DELAY: None,
+            ATTR_REAL_TIME: None,
+            ATTR_DESTINATION: None,
```

Comment:
When I check the python library, in the readme it states that if something is unavailable or has an error, it returns "n/a" as well. So this change would only fix the initialization of the sensor, but when an error occurs (or probably some other event happens) it will still return "n/a". So maybe this should be fixed in the library first before fixing it here.

Reply:
Good catch, I've updated the logic to consider and handle this case. The HA integration is currently broken as it does not handle the response "n/a" from the lib inline with what HA expects. The context of why the lib responds in the way it does would be for the maintaners of that lib to determine.